Dockerfile

```diff
@@ -18,12 +18,16 @@ RUN service mysql start \
     FLUSH PRIVILEGES;" \
  && mysql -u "root" -e "CREATE DATABASE IF NOT EXISTS local_reporter;"
 # write environment variables
+ARG AIMLAC_API_KEY="" \
+    AIMLACKIES_REPORTER_SECRET_KEY="W5Jal+qx9vOh8gjaugNZPg==" \
+    AIMLACKIES_REPORTER_SECURITY_PASSWORD_SALT="288872824872550364092124909785067926279" \
+    AIMLACKIES_REPORTER_DATABASE_URL="mysql+pymysql://user0:secret@localhost/local_reporter"
 RUN echo -n \
 "export AIMLAC_CC_MACHINE=34.72.51.59\n\
-export AIMLAC_API_KEY=AIMLACkies275001901\n\
-export AIMLACKIES_REPORTER_SECRET_KEY=W5Jal+qx9vOh8gjaugNZPg==\n\
-export AIMLACKIES_REPORTER_SECURITY_PASSWORD_SALT=288872824872550364092124909785067926279\n\
-export AIMLACKIES_REPORTER_DATABASE_URL=mysql+pymysql://user0:secret@localhost/local_reporter\n\
+export AIMLAC_API_KEY=${AIMLAC_API_KEY}\n\
+export AIMLACKIES_REPORTER_SECRET_KEY=${AIMLACKIES_REPORTER_SECRET_KEY}\n\
+export AIMLACKIES_REPORTER_SECURITY_PASSWORD_SALT=${AIMLACKIES_REPORTER_SECURITY_PASSWORD_SALT}\n\
+export AIMLACKIES_REPORTER_DATABASE_URL=${AIMLACKIES_REPORTER_DATABASE_URL}\n\
```

Comment:
I've made these 4 variables as build arguments, please include `--build-arg AIMLAC_API_KEY=XXX` when building the image